modules/ossm-mt-installation.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-mt-ossm.adoc
+
+[id="ossm-mt-installation_{context}"]
+= Installing {ProductName} with multi-tenancy
+
+This procedure describes the how to enable multi-tenancy in your custom resource file before deploying the control plane:
+
+.Prerequisites
+
+* An installed, verified {ProductShortName} operator.
+* A custom resource file that defines the parameters of your {ProductName} control plane.
+
+.Procedure
+
+Set the `multitenant` option to `true` in the istio section of the `ServiceMeshControlPlane` resource. For example,
```

Comment:
Single step procedures need to be presented as an unordered list, so s/Set/. Set

Reply:
Fixed.